src/App.tsx

```diff
@@ -3,33 +3,58 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-type State = {};
+type State = {
+  query: string;
+};
 
 export class App extends React.Component<{}, State> {
-  state: State = {};
+  state: State = {
+    query: '',
+  };
+
+  filterMovies = () => {
```

Comment:
```suggestion
  getFilterMovies = () => {
```

find a book
get a movie

verbNoun
findBook
getMovie